README.md

```diff
@@ -1,33 +1,35 @@
+| BIND_ADDR                        | :29200                 | The host and port to bind to                                                                                                            |
+| GRACEFUL_SHUTDOWN_TIMEOUT        | 5s                     | The graceful shutdown timeout in seconds (`time.Duration` format)                                                                       |
+| HEALTHCHECK_INTERVAL             | 30s                    | Time between self-healthchecks (`time.Duration` format)                                                                                 |
+| HEALTHCHECK_CRITICAL_TIMEOUT     | 90s                    | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)                      |
+| OTEL_BATCH_TIMEOUT               | 5s                     | Time duration after which a batch will be sent regardless of size (`time.Duration` format)                                              |
+| OTEL_EXPORTER_OTLP_ENDPOINT      | localhost:4317         | OpenTelemetry Exporter address                                                                                                          |
+| OTEL_SERVICE_NAME                | dp-legacy-cache-proxy  | The name of this service in OpenTelemetry                                                                                               |
+| BABBAGE_URL                      | http://localhost:8080  | Babbage address, where all the incoming requests are forwarded to                                                                       |
+| LEGACY_CACHE_API_URL             | http://localhost:29100 | Legacy Cache API address                                                                                                                |
+| CACHE_TIME_DEFAULT               | 15m                    | Default value for the `max-age` directive of the `Cache-Control` header (`time.Duration` format)                                        |
+| CACHE_TIME_ERRORED               | 30s                    | Errored value for the `max-age` directive of the `Cache-Control` header (`time.Duration` format)                                        |
+| CACHE_TIME_LONG                  | 4h                     | Long value for the `max-age` directive of the `Cache-Control` header (`time.Duration` format)                                           |
+| CACHE_TIME_SHORT                 | 10s                    | Short value for the `max-age` directive of the `Cache-Control` header (`time.Duration` format)                                          |
+| IS_PUBLISH_EXPIRY_OFFSET_ENABLED | false                  | Determines if publish expiry offset is used which enables a shorter cache time for recently published content.                          |
```

Comment:
not a big one, but I would have preferred the more usual pattern `ENABLE_PUBLISH_EXPIRY_OFFSET` format

Reply:
Have updated - thanks gedge